critiquebrainz/db/review.py

```diff
@@ -40,6 +40,21 @@ def to_dict(review, confidential=False):
     return review
 
 
+def get_top_languages():
+    with db.engine.connect() as connection:
+        result = connection.execute(sqlalchemy.text("""
+        SELECT  language, 
+                COUNT(id) AS count
+        FROM review
+        GROUP BY language
+        ORDER BY count
+        DESC LIMIT 10
+        """))
+    languages = result.fetchall()
```

Comment:
`languages = result.fetchall()` must be within `with db.engine.connect() as connection` indent. (connection is closed when the `with` code block is left.